testing/test_browser.py

```diff
@@ -218,3 +218,67 @@ def test_size(browser):
     width, height = browser.size_of('#exact_dimensions')
     assert width == 42
     assert height == 69
+
+
+def test_current_window_handle(browser):
+    assert browser.title == "Test page"
```

Comment:
Please write some docblocs for these tests, stating what component of window handling they're covering.

This case looks more like a test of the new title attribute than it is a test of current_window_handle, which is tested in test_window_handles below anyway.

Reply:
Added separate tests for each method... with small doc block. :+1: